src/assets/index.js

```diff
@@ -1 +1,4 @@
 export { default as landing_image } from "./images/landing_image.jpg";
+export { default as contact_header } from "./images/contact_header.jpg";
```

Comment:
Seems you might have cloned from CDS-7, could you cherry-pick the one commit you want onto master?

Reply:
And through some squashing, it's resolved lol.